deploy/infrastructure/modules/terraform-google-dss/README.md

```diff
@@ -0,0 +1,70 @@
+# terraform-google-dss
+
+This terraform module creates a Kubernetes cluster in Google Cloud Engine and generates 
+the tanka files to deploy a DSS instance.
+
+## Getting started
+
+### Prerequisites
+Download & install the following tools to your workstation:
+
+1. Install [terraform](https://developer.hashicorp.com/terraform/downloads).
+2. Install tools from [Prerequisites](../../../../build/README.md)
+3. Install provider specific tools:
+    1. [Google Cloud Engine](./README.md#google-cloud-engine)
+    
+#### Google Cloud Engine
+
+1. Install and initialize [Google Cloud CLI](https://cloud.google.com/sdk/docs/install-sdk).
+    1. Confirm successful installation with `gcloud version`.
+2. Check that the DSS project is correctly selected: gcloud config list project
+    1. Set another one if needed using: `gcloud config set project $GOOGLE_PROJECT_NAME`
+3. Enable the following API using [Google Cloud CLI](https://cloud.google.com/endpoints/docs/openapi/enable-api#gcloud):
+    1. `container.googleapis.com`
+    2. If you want to manage DNS entries with terraform: `dns.googleapis.com`
+4. Install the auth plugin to connect to kubernetes: `gcloud components install gke-gcloud-auth-plugin`
+5. Run `gcloud auth application-default login` to generate credentials to call Google Cloud Platform APIs.
+
+### Deployment of the Kubernetes cluster
+
+1. Create a new folder in `/deploy/infrastructure/personal/` named for instance `terraform-google-dss-dev`.
+2. Copy main.tf, output.tf and variables.tf to the new folder. (Note that the modules can be added to existing projects)
+3. Copy `terraform.dev.example.tfvars` or `terraform.example.tfvars`, rename to `terraform.tfvars` and set the variables according to your environment.
+4. Check that your new directory contains the following files:
+   - main.tf
+   - output.tf
+   - variables.tf
+   - terraform.tfvars
+5. Initialize terraform: `terraform init`.
+6. Run `terraform plan` to check that the configuration is valid. It will display the resources which will be provisioned.
+7. Run `terraform apply` to deploy the cluster. (This operation may take up to 15 min.)
+8. Configure the DNS resolution to the public ip addresses. DNS entries can be either managed manually or 
+handled by terraform depending on the cloud provider. See [DNS](DNS.md) for details.
+
+## Deployment of the DSS services
+
+During the successful run, the terraform job has created a new [workspace](../../../../build/workspace/)
+for the new cluster.
+
+It contains scripts to operate the cluster and setup the services.
+
+1. Go to `/build/workspace/${CLUSTER_CONTEXT}`.
```

Comment:
How should a user determine their ${CLUSTER_CONTEXT}?

Reply:
Added the information to terraform output and explanation in README.

Reply:
I'll add a new comment to the new line and resolve this conversation.